content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -241,12 +229,47 @@ getting replaced.
 
 Due to the nature of EndpointSlice changes, endpoints may be represented in more
 than one EndpointSlice at the same time. This naturally occurs as changes to
-different EndpointSlice objects can arrive at the Kubernetes client watch/cache
-at different times. Implementations using EndpointSlice must be able to have the
-endpoint appear in more than one slice. A reference implementation of how to
-perform endpoint deduplication can be found in the `EndpointSliceCache`
-implementation in `kube-proxy`.
+different EndpointSlice objects can arrive at the Kubernetes client watch / cache
+at different times, and is a design detail rather than a bug.
```

Comment:
```suggestion
at different times.
```

Reply:
OK - but why?

Reply:
Imo it's because in general, the docs always document what the product does because it was designed to do the thing. If we did have a limitation or a bug, we'd explicitly say so. If the feature or product is performing as designed, we don't need to explicitly call that out, because that's the whole idea of documenting that

If we say "it's a feature, not a bug" (lol), almost sounds like we're trying to justify a bad design decision (which it might not have been at all!)